pkg/reddit/fetch.go

```diff
@@ -246,10 +246,10 @@ func getPost(postUrl string, root map[string]interface{}) (fetchResult interface
 				if strings.HasPrefix(root["url"].(string), "https://i.imgur.com") { // Example: https://www.reddit.com/r/dankmemes/comments/gag117/you_daughter_of_a_bitch_im_in/
 					gifDownloadUrl := root["url"].(string)
 					lastSlash := strings.LastIndex(gifDownloadUrl, "/")
-					gifDownloadUrl = gifDownloadUrl[:lastSlash+1] + "download" + gifDownloadUrl[lastSlash:]
+					gifDownloadUrl = gifDownloadUrl[:lastSlash+1] + "Download" + gifDownloadUrl[lastSlash:]
```

Comment:
I think this one must be in lowercase as well

Reply:
Sure.